### DISCUSSION TITLE
Possible instruction error in Basic math in JavaScript Guide?

### DISCUSSION THREAD

**Discussion by iamvictati** [mdn/mdn-community#703] [Content] [answered]
Hi, 

I'm quite new at this and currently completing TOP so I apologize for my ignorance or perhaps boldness (lol) but I came across an instruction in the [Active learning: sizing a canvas box](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/First_steps/Math#active_learning_sizing_a_canvas_box) section that seems a bit mathematically weird for me. 

Link: [Editable Canvas Example](https://mdn.github.io/learning-area/javascript/introduction-to-js-1/maths/editable_canvas.html)

Exercise 3 says: 

"Change the line that calculates x so the box is 250px wide, but the 250 is calculated using two numbers and the remainder (modulo) operator." 

```
// Edit the two lines below here ONLY
x = 50;
y = 50;

ctx.fillStyle = 'green';
ctx.fillRect(10, 10, x, y);
```

How exactly is one supposed to calculate a total remainder of exactly 250px using % and only 2 numbers? Or was this instruction meant to say “division operator”? A division seems to make more sense, given that it's simple JS math. 

Once again, apologies if I'm overstepping (lol) but I wasn't able to let it go and the only simple solution I found was to state `const boxWidth = 250 + remainder` which isn't part of the code given in the example.

**Comment by Josh-Cena** [comment]
You can calculate `50` using `43` and `7` using `+`. Surely you can create an expression that produces `250` using two numbers and `%`?

**Reply by iamvictati** [reply]
Thanks for your reply Josh. Could you please give the math expression? Math isn't really my thing and I can't find the answer with simple mental math.

**Reply by Josh-Cena** [reply, marked as answer]
Sure. Your goal is to find `a` and `b` such that `a % b = 250`. In other words, `a = b * c + 250`, where `b` is something larger than 250 (so it can allow a remainder of 250). Therefore one easy solution is to pick `b = 251` and `c = 1`, then you would have `a = 251 + 250 = 501`. Verify: `501 % 251 === 250`.

**Reply by iamvictati** [reply]
Thank you, Josh!!! :))) 

The relief! I asked the AI for help and it gave me crazy answers with more complex math formulas and I couldn't let it go haha. Thanks for the save! 

I think the exercise should use a simple division though for learning’s sake. I couldn't figure out the math by myself. 

Either way, you are a saviour I really appreciate it! ^.^

**Reply by Josh-Cena** [reply]
No problem! I wouldn't trust AI on math-related stuff. In the future, if you have problems with MDN content, you can also ask in our [Discord server](https://developer.mozilla.org/discord), where it's hopefully more discoverable.

**Reply by iamvictati** [reply]
Ok, I'll keep that in mind and already joined the server. :)))
